examples/fizzbuzz/README.md

```diff
@@ -0,0 +1 @@
+# fizzbuzz
```

Comment:
@Jagl257 Could you please add a little bit more of description? I think we can include some of the description of the problem and then the steps to execute it.

Reply:
Ok @CristhianMotoche, can you check them out, please?